doc/source/whatsnew/v1.0.0.rst

```diff
@@ -89,7 +89,7 @@ Categorical
 ^^^^^^^^^^^
 
 - Added test to assert the :func:`fillna` raises the correct ValueError message when the value isn't a value from categories (:issue:`13628`)
--
+- Bug in merge on CategoricalIndex fails if left_index=True & right_index=True, but not if on={index} (:issue:`28189`)
```

Comment:
Put code like left_index=True in double backticks.

What does `on={index}` mean?

Reply:
It's the case that work on the example of the issue! #28189